pkg/reconciler/broker/controller.go

```diff
@@ -124,16 +135,3 @@ func newController(ctx context.Context, cmw configmap.Watcher, drs *dataresidenc
 
 	return impl
 }
-
```

Comment:
It looks like we deleted this helper method and put the logic to each caller? Instead of deleting this helper method, can we put it in a common package so that everyone can call it and we also can get rid of the duplicate?

Reply:
That's what I was trying to do at the beginning, as I wanted to have sth similar to the sources as well. The original code actually called `utils.ProjectID` already, so this helper function is not really useful at all. Also, this helper function is too small and not generic enough to be moved to a common place.